Packs/GitLab/ReleaseNotes/2_2_23.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### GitLab v2
+
+- Fixed an issue in ***gitlab-commit-list*** command where *created_after* and *created_before* didn't filter commits properly.
```

Comment:
```suggestion
- Updated the ***gitlab-commit-list*** command to use *since* instead of *created_after* and *until* instead of *created_before* due to API changes.
```